### DISCUSSION TITLE
Configure security context for Pods and containers when using Helm chart

### DISCUSSION THREAD

**Discussion by johnnycederholm** [seaweedfs/seaweedfs#5598] [General]
I recently found this project and have started to testing it by deploying it to a Kubernetes cluster that I have, using the Helm chart found in this repo. When deploying Seaweed I would like to configure the security context for the Pods and containers which doesn't seem to be supported at the moment with the most recent Helm chart.

If this capability is missing would you be interested in a PR adding it?

**Comment by chrislusf** [comment]
PR is very welcome!

**Comment by johnnycederholm** [comment]
Thank you for your fast response! I created a PR #5600 to add the possibility to set the security context for pod and containers when deploying with the Helm chart.
